package.json

```diff
@@ -21,29 +25,25 @@
     "webpack": "^4.41.2"
   },
   "devDependencies": {
+    "@babel/core": "^7.0.0-0",
```

Comment:
Latest is `7.7.4`.  Should we use the latest?

Reply:
Bumped.